lldb/include/lldb/Target/Target.h

```diff
@@ -310,6 +310,10 @@ class TargetProperties : public Properties {
   bool GetEnableTrampolineSupport() const;
 
 private:
+  std::optional<bool>
+  GetExperimentalPropertyValue(size_t idx,
```

Comment:
Should this function live upstream?

Reply:
yep. (on first glance I thought these were all swift, but one property, `inject-local-vars`, does exist upstream).

Reply:
I'll pick this change into this branch: https://github.com/llvm/llvm-project/pull/83504